cdc/server.go

```diff
@@ -350,3 +360,119 @@ func (s *Server) Close() {
 		s.statusServer = nil
 	}
 }
+
+func (s *Server) setUpDataDir(ctx context.Context) error {
+	conf := config.GetGlobalServerConfig()
+	if conf.DataDir != "" {
+		conf.Sorter.SortDir = filepath.Join(conf.DataDir, config.DefaultSortDir)
+		config.StoreGlobalServerConfig(conf)
+
+		return nil
+	}
+
+	// data-dir will be decide by exist changefeed for backward compatibility
```

Comment:
Do we have a design doc for this? And what if we don't have a changefeed initially, but the user plans to create one soon, whose `sort-dir` is what the user intends for TiCDC to use?